frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -432,22 +448,29 @@ export const WhatsNewMenu = (props: Props) => {
   const phoneAnnouncement: WhatsNewEntry = {
     title: l10n.getString("whatsnew-feature-phone-header"),
     snippet: l10n.getString("whatsnew-feature-phone-snippet"),
-    content: (
-      <WhatsNewContent
-        description={l10n.getString("whatsnew-feature-phone-description")}
-        heading={l10n.getString("whatsnew-feature-phone-header")}
-        image={PhoneMaskingHero.src}
-        videos={{
-          // Unfortunately video files cannot currently be imported, so make
-          // sure these files are present in /public. See
-          // https://github.com/vercel/next.js/issues/35248
-          "video/webm; codecs='vp9'":
-            "/animations/whatsnew/phone-masking-hero.webm",
-          "video/mp4": "/animations/whatsnew/phone-masking-hero.mp4",
-        }}
-        cta={ctaUpgradePhones}
-      />
-    ),
+    content:
+      props.runtimeData && isPhonesAvailableInCountry(props.runtimeData) ? (
+        <WhatsNewContent
+          description={l10n.getString("whatsnew-feature-phone-description")}
+          heading={l10n.getString("whatsnew-feature-phone-header")}
+          image={PhoneMaskingHero.src}
+          videos={{
+            // Unfortunately video files cannot currently be imported, so make
+            // sure these files are present in /public. See
+            // https://github.com/vercel/next.js/issues/35248
+            "video/webm; codecs='vp9'":
+              "/animations/whatsnew/phone-masking-hero.webm",
+            "video/mp4": "/animations/whatsnew/phone-masking-hero.mp4",
+          }}
+          cta={
+            <CtaLinkButton
+              subscribed={props.profile.has_phone}
+              link={getPhoneSubscribeLink(props.runtimeData, "monthly")}
```

Comment:
To avoid having to choose a specific billing period for the user here, [Eduardo suggested](https://mozilla.slack.com/archives/C013CSYEL5T/p1662565686526129?thread_ts=1662548886.744219&cid=C013CSYEL5T) making this link point to `/premium#pricing` instead. (Which will need [a Next.js `<Link>`](https://nextjs.org/docs/routing/introduction#linking-between-pages) instead of an `<OutboundLink>`.)

Reply:
done.